index.html

```diff
@@ -2,50 +2,50 @@
 
 <html lang="fi">
     <head>
-        <meta charset="UTF-8">
-        <meta name="viewport" content="width=device-width, initial-scale=1">
-        
-        <meta name="description" content="Kaikkien TTY:n opiskelijaravintoloiden ruokalistat">
-        <meta name="theme-color" content="#222222">
-        
+        <meta charset="UTF-8" />
```

Comment:
No point adding these

Reply:
They were automatically added by my linter sry for that. Although if it's so important maybe the project could have contained a eslintrc or prettierrc to implicate the importance.